Tables/FE8 Nightmare modules/FE8 Personal Skill Editor.csv

```diff
@@ -30,7 +30,7 @@ Dozla,PivotID
 Rennac,SpurSpdID
 Duessel,RepositionID
 Myrrh,DragonsBloodID
-Knoll,VoiceOfPeaceID
+Ersatz,DrawBackID
```

Comment:
DrawBackID is undefined. Draw Back is not implemented in the skill system. 

Review Engine Hacks\Skill System\skill_definitions.event for available skills.